packages/block-library/src/file/edit.native.js

```diff
@@ -70,6 +95,29 @@ export default class FileEdit extends Component {
 		this.props.setAttributes( { downloadButtonText } );
 	}
 
+	onChangeDownloadButtonVisibility( showDownloadButton ) {
+		this.props.setAttributes( { showDownloadButton } );
+	}
+
+	onCopyURL() {
+		if ( this.state.isUrlCopied ) {
+			return;
+		}
+		const { href } = this.props.attributes;
+		Clipboard.setString( href );
+
+		this.setState( { isUrlCopied: true } );
+		this.timerRef = setTimeout( () => {
+			this.setState( { isUrlCopied: false } );
+		}, 1500 );
```

Comment:
@etoledom is it okay to have this value hardcoded here or do we want to put it in a constant? Whichever works for me. WDYT?

Reply:
Done in https://github.com/WordPress/gutenberg/pull/26360/commits/53a20651d130de3195faadbdddfb796782c4c091
Nice catch!